modules/cache/redis/main.bicep

```diff
@@ -94,6 +94,12 @@ param subnetId string = ''
 @description('Optional. A dictionary of tenant settings.')
 param tenantSettings object = {}
 
+@description('Optional. When true, replicas will be provisioned in availability zones specified in zones param. When false, service will choose where replicas are deployed.')
+param zoneRedundant bool = true
```

Comment:
@lsnoddy can you please test those parameters in the common test file?

Reply:
Added zoneRedundant and zones to the common test file. Thanks for the reminder.